test/performance/odp_pool_latency.c

```diff
@@ -547,14 +556,9 @@ static parse_result_t check_options(prog_config_t *config)
 		return PRS_NOK;
 	}
 
-	if (config->num_rounds == 0U) {
-		ODPH_ERR("Invalid round count: %u (min: 1)\n", config->num_rounds);
-		return PRS_NOK;
-	}
-
-	if (config->num_ignore >= config->num_rounds) {
-		ODPH_ERR("Invalid round ignorance count: %u (max: %u)\n", config->num_ignore,
-			 config->num_rounds - 1U);
+	if (config->num_rounds > 0U && config->num_ignore >= config->num_rounds) {
+		ODPH_ERR("Invalid round ignorance count: %" PRIu64 " (max: %" PRIu64 ")\n",
```

Comment:
Typo: ignorance -> ignore. Same also in the commit message.

Reply:
Well, I wanted use a noun and that's the noun of "ignore".